R/parseTimeseries.R

```diff
@@ -0,0 +1,45 @@
+
+#'@title parse timeseries file into R environment
+#'
+#'@details a function for loading data into R from a file (or URL) from a completed processing request
+#'
+#'@param file a \code{\link{geoknife}} timeseries processing result file location
+#'(See \code{\link{checkProcess}}).
+#'@param delim the file delimiter
+#'@return list of timeseries values.
+#'@keywords methods
+#'@author Luke A. Winslow, Jordan S. Read
+#'@export
+#'@examples
+#'local_file <- system.file('extdata','tsv_linear_ring.tsv', package = 'geoknife')
+#'output <- parseTimeseries(local_file, delim = '\t')
+#'@importFrom lubridate parse_date_time2
+parseTimeseries <- function(file, delim){
```

Comment:
yeah, I am going to add another issue for handling the more complex timeseries parsing, including more than one stat and more than on variable. Anything else than can make the .csv trickier? I found earlier than in some datasets (NLDAS) some variables may have 1 fewer timestep on leapyears. So, you also can't assumed the blocks are the same length.